src/main/cms/cms.c

```diff
@@ -82,6 +82,8 @@
 
 displayPort_t *pCurrentDisplay;
 
+unsigned int osd_profile_cursor = 1;
```

Comment:
camelCase, `static`, and it should be probably moved closer to `cmsCtx_t`

Reply:
Fixed.